app/helpers/textual_mixins/textual_paused_state.rb

```diff
@@ -0,0 +1,12 @@
+module TextualMixins::TextualPausedState
+  def textual_paused_state
+    state = @record.enabled?
+    h = {:label => _("Running State"), :value => state ? _("Running") : _("Paused")}
+    if state
```

Comment:
Good point... this will be really confusing... it should be: "Collection Status" @serenamarie125  Idea?